### ISSUE TITLE
project abadoned?

### ISSUE THREAD

**Issue by CodeAnthem** [chefbennyj1/Emby.IntroSkip.Release#10]
Hi 

I was looking for an update and noticed the last release is older than the version I currently have.
Did this project move or is it abadoned?

**Comment by noir1216** [comment]
Hi,Can you please send me a copy of the version you are currently using?

**Comment by chefbennyj1** [comment]
Emby actually used some of this code, and integrated their own intro skip. 
This project is abandoned 😃

**Comment by noir1216** [comment]
> Emby actually used some of this code, and integrated their own intro skip. This project is abandoned 😃

Emby own intro skip have too many recognition errors and are not easy to manage at a later stage, with no way to manually filter to remove mis-recognized episode

**Comment by chefbennyj1** [comment]
Yeah, this project did have that option. You could probably load the project, and do a compatibility update to get it working. But, you wouldn't have access to the skip intro button in emby. Or at least you wild have to figure out where emby is as saving the data and learn how to edit their data with this plugin UI.
